.soliumrc.json

```diff
@@ -4,6 +4,9 @@
     "security"
   ],
   "rules": {
+    "security/no-block-members": [
+      "off"
```

Comment:
Why are we turning off this lint?

Reply:
Because we must use the now command.
The ideal thing would be to turn it off just for a specific file or line, but we couldn't find if it is even possible.

Reply:
I found a way..will update soon.

Reply:
Fixed at a2c5a54